plugins/scaffolder/src/components/fields/GroupsImPartOfPicker/GroupsImPartOfPicker.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { identityApiRef, useApi } from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  TextField,
+  MenuItem,
+  Select,
+  FormControl,
+  InputLabel,
+} from '@material-ui/core';
+import {
+  GroupsImPartOfPickerProps,
+  GroupsImPartOfPickerSchema,
+} from './schema';
+
+export { GroupsImPartOfPickerSchema };
+
+export const GroupsImPartOfPicker = (props: GroupsImPartOfPickerProps) => {
```

Comment:
Still a little unsure about this name :joy: it could be something like `OwnershipEntityRefPicker` or something. Do you have any thoughts?

Reply:
I'm terrible with names 😂, `OwnershipEntityRefPicker` this sounds good to me 👍